src/app/lib/seoUtils/getLiveBlogPosting/index.ts

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+import buildIChefURL from '#app/lib/utilities/ichefURL';
+import { OptimoBlock } from '#app/models/types/optimo';
+import { Post } from '../../../../../ws-nextjs-app/pages/[service]/live/[id]/Post/types';
+
+export default ({
+  posts,
+  brandName,
+  defaultImage,
+  url,
+}: {
+  posts?: Post[];
+  brandName: string;
+  defaultImage: string;
+  url: string;
+}) => {
+  if (!posts) return null;
+
+  return {
+    liveBlogPosting: posts
+      .map(result => {
+        const headerBlocks = result?.header?.model?.blocks;
+        const contentBlocks = result?.content?.model?.blocks;
+
+        if (!headerBlocks || !contentBlocks) return null;
+
+        const headlineBlock = headerBlocks.find(
+          block => block.type === 'headline',
+        );
+
+        const paragraphBlocks = contentBlocks.filter(
+          block => block.type === 'paragraph',
+        );
+
+        const imageBlock = contentBlocks.find(block => block.type === 'image');
+
+        // @ts-ignore - deeply nested
```

Comment:
Could we use `ts-expect-error` instead of `ts-ignore`? (by doing so, if it is ever fixed / not a problem anymore, then the ts compile will fail, and it will ask us to remove the `ts-expect-error`. It means you can also remove `/* eslint-disable @typescript-eslint/ban-ts-comment */` from the top of the file)

Reply:
Yes! Good shout.